app/controllers/events_controller.rb

```diff
@@ -0,0 +1,34 @@
+class EventsController < ApplicationController
+  include EventsHelper
+
+  def new
+    @event = Event.new
+  end
+
+  def create
+    @event = User.find_by(id: session[:user_id]).events.build(event_params)
```

Comment:
##### EventsController

###### `create` action

* [ ] uses #build method on association, i.e. `current_user.events.build(event_params)` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/build_method_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))

- Create the current_user variable so you don't have to do this `User.find_by(id: session[:user_id])` several times